recipes/environ-config/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "environ-config" %}
+{% set version = "21.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/environ-config-{{ version }}.tar.gz
+  sha256: d82fa2f760ac031fb9f434be56a0a83e98dbeba8151cdd96feaf86cd17417dd9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Need to also have `python >=3.3` (though I think the oldest conda-forge even has is 3.5)